### DISCUSSION TITLE
Modifying GitHub Desktop 's shortcuts

### DISCUSSION THREAD

**Discussion by eai7** [community/community#24158] [Other Feature Feedback, Questions, & Ideas]
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/keyboard-shortcuts" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/keyboard-shortcuts" target="_blank" rel="noopener nofollow ugc">Keyboard shortcuts - GitHub Docs</a></h3>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<blockquote>
<p><em>Is there a way to modify/edit/change Github Desktop shortcuts?</em></p>
</blockquote>
<blockquote>
<p><strong>1.</strong> Is there a way to change the keys for a shortcut corresponding to a function?</p>
</blockquote>
<pre><code class="lang-auto">e.g.
Ctrl+Shift+A ---&gt; Ctrl+Shift+C  
to Open the repository in your preferred editor tool
</code></pre>
<blockquote>
<p><strong>2.</strong> Is there a way to make Github Desktop read/use/talk with .gitignore while openin a repository with my preferred editor tool?</p>
</blockquote>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/ordinaryusrex">@OrdinaryusRex</a> Thanks for the questions.</p>
<blockquote>
<p><strong>1.</strong>  Is there a way to change the keys for a shortcut corresponding to a function?</p>
</blockquote>
<p>Currently it is not possible to modify the available keyboard shortcuts in GitHub Desktop, and at this time we do not have any plans to enable this. If there is a specific shortcut you would like to see added please feel free to open an issue in <a href="https://github.com/desktop/desktop/issues">https://github.com/desktop/desktop/issues</a>.</p>
<blockquote>
<p><strong>2.</strong>  Is there a way to make Github Desktop read/use/talk with .gitignore while openin a repository with my preferred editor tool?</p>
</blockquote>
<p>I’m not sure I fully understand the question, but GitHub Desktop will respect your existing .gitignore file and you can also ignore files directly in GitHub Desktop by right-clicking on them in the <code>Changes</code> view. Let me know if I’m misunderstanding your question. Thanks for using GitHub Desktop!</p>

**Comment by impratikjaiswal** [comment]
<p>Hello <a class="mention" href="/steveward">@steveward</a> ,</p>
<p>Since you clarified that we can not modify keys for a keyboard shortcuts,<br>
Is it possible to disable / block a specific shortcut or all keyboard shortcuts in github desktop</p>
<p>for e.g.:<br>
I would like to disable Ctrl + Shift + S (Stash all changes) as i am using this shortcut in various other editiors for “show in explorer” fuction.</p>

**Comment by steveward** [comment]
<p><a class="mention" href="/impratikjaiswal">@impratikjaiswal</a> there isn’t a way to block shortcuts in GitHub Desktop, but there are applications that you can use to modify global shortcuts on your system. Is that what you are using?</p>
